.gitignore

```
@@ -96,3 +96,6 @@ pg/postgres-data
 
 # rollup visualizer output
 frontend/stats.html
+
+# timeline SVG
+frontend/public/assets/gemRepository/integratedModelsTimeline.svg
```

I would opt for a more robust, filename-agnostic ignore rule:

```suggestion
frontend/public/assets/gemRepository/*.svg
```

---

I don't see the need of this since we don't know what we might want to track or not in the future

---

I don't have too strong opinion for this specific case. In general I agree with @inghylt in the sense that I don't want to anticipate too much about what might or might not happen in the future. This is not quite [premature optimization](https://stackify.com/premature-optimization-evil/) but the approach is similar.